extensions/amp-pan-zoom/amp-pan-zoom.md

```diff
@@ -133,3 +133,7 @@ You can also hide these buttons entirely and create your own using the `transfor
 
 ## Validation
 See [amp-pan-zoom rules](https://github.com/ampproject/amphtml/blob/master/extensions/amp-pan-zoom/validator-amp-pan-zoom.protoascii) in the AMP validator specification.
+
+## Eligibile children tags
+See the [list](https://github.com/ampproject/amphtml/blob/e517ee7e58215ea8baaa04fa5c6b09bba9581549/extensions/amp-pan-zoom/0.1/amp-pan-zoom.js#L47) of eligibles children tags of `amp-pan-zoom`.
+In order to correcly display images, SVGs need the default `meet` value for the `preserveAspectRatio` attribute.
```

Comment:
We can take out this line. It doesn't have a whole lot to do with `<amp-pan-zoom>`.